### PR TITLE
[connectors] - refactor(google_drive): streamline `file.ts`

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR aims at refactoring the `google_drive/temporal/file.ts` file to split the `syncOneFile` function into sub-functions for readability and modularity purposes.

## Risk

Breaking existing behaviour

## Deploy Plan

Deploy `connectors`

### CODE REVIEW

File: connectors/package-lock.json [hunk 1]
```diff
@@ -12360,4 +12360,4 @@
       }
     }
   }
-}
+}
```
Comment (flvndvd) [inline connectors/package-lock.json:12363]: No need for this change, right?

File: connectors/src/connectors/google_drive/temporal/file.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ import tracer from "dd-trace";
 import fs from "fs/promises";
 import type { OAuth2Client } from "googleapis-common";
 import os from "os";
+import type { Logger } from "pino";
```
Comment (flvndvd) [inline connectors/src/connectors/google_drive/temporal/file.ts:8]: We should import from `@connectors/logger/logger`.

File: connectors/src/connectors/google_drive/temporal/file.ts [hunk 3]
```diff
@@ -40,6 +41,260 @@ import logger from "@connectors/logger/logger";
 import type { DataSourceConfig } from "@connectors/types/data_source_config";
 import type { GoogleDriveObjectType } from "@connectors/types/google_drive";
 
+async function handleGoogleDocExport(
```
Comment (flvndvd) [inline connectors/src/connectors/google_drive/temporal/file.ts:44]: This looks great 🔥 Can you rebase on the latest changes with Tika 🙏

**Review by flvndvd — COMMENTED** [review]
Sorry for the late review 🙈

File: connectors/src/connectors/google_drive/temporal/file.ts (hunk 3 above)
Comment (lasryaric) [reply connectors/src/connectors/google_drive/temporal/file.ts:45]: This function should be called `handleGoogle[Drive]Export()`. The Google doc API is a different one.

Comment (lasryaric) [reply connectors/src/connectors/google_drive/temporal/file.ts:45]: Or for consistency with `handleFileDownload()`, we can also call it `handleFileExport()`

**Review by lasryaric — COMMENTED** [review]
Left 2 little comments. Looks much much better than what we have today 🙏

File: connectors/src/connectors/google_drive/temporal/file.ts (hunk 3 above)
Comment (JulesBelveze) [reply connectors/src/connectors/google_drive/temporal/file.ts:45]: ✅ Used Tika for pptx, .docx and pdf

**Review by flvndvd — APPROVED** [review]
Left a comment around a dependency that should not be there. Assuming you've tested.

**Pull request merged**